forte/mrdsrg-spin-integrated/dsrg_mrpt2_grad/dsrg_mrpt2_deriv_multipliers.cc

```diff
@@ -0,0 +1,383 @@
+/**
+ * Solve several multipliers directly.
+ */
+#include "psi4/libpsi4util/PsiOutStream.h"
+#include "../dsrg_mrpt2.h"
+#include "helpers/timer.h"
+
+using namespace ambit;
+using namespace psi;
+
+namespace forte {
+
+void DSRG_MRPT2::set_tau() {
+    outfile->Printf("\n    Initializing multipliers for two-body amplitude.. ");
+    Tau2 = BTF_->build(CoreTensor, "Tau2", {"hhpp", "hHpP"});
+    // Tau * Delta
+    // <[V, T2]> (C_2)^4
+    if (PT2_TERM) {
```

Comment:
Shall we have a more detailed explanation of what these PT2_TERM, X1_TERM, ... are?

Reply:
I have added detailed eqs associated to these parameters in the .h file.